properties/handlers.py

```diff
@@ -6,10 +6,16 @@
 
 from six import string_types
 
+from .utils import everything
+
 
 def _set_listener(instance, obs):
     """Add listeners to a Properties class instance"""
-    for name in obs.names:
+    if obs.names is everything:
+        names = list(instance._props)
```

Comment:
Hmm, I think I might disagree here. I don't think this behaviour is too magic; it is the python-future recommended way to make a list of keys (http://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list). Additionally, it is assigned to `names` - maybe that will cue someone who is unclear that this is a list of names?